schemas/manifest.fbs

```diff
@@ -5,21 +5,30 @@ table Uuid {
     low: uint64;
 }
 
-table DbParent {
-    // Path to the parent database from which this database was cloned
+// Reference to an external database.
+table ExternalDb {
+    // Path to root of the external database
     path: string (required);
 
-    // Parent checkpoint ID that this clone was derived from
-    checkpoint: Uuid (required);
+    // Externally owned Checkpoint ID we've used to create an initial state of cloned database.
+    source_checkpoint_id: Uuid (required);
```

Comment:
I recall you and @rodesai discussing this, but not sure where we landed: is a source checkpoint required if I want to checkpoint against the latest manifest? If so, is there always going to be a checkpoint for the latest manifest?